### PR TITLE
Created project Readme.md & Removed unused items

### PR DESCRIPTION
**Pull request by BT-Creator**
This pull request adds a README.md file with the following contents:
- Introduction
- An deployment guide for local development
- A reference to the server deployment guide
- Credits to all team members
- Link to the engineering wiki

This PR also removes some unused project in `misc` that were used to get familiar with the tech.

I was supposed to also reorganize the repo a little bit, but it seems that the Docker gods were against me, so I'm keeping my hands of it for now.

### CODE REVIEW

**Review by jodiDL — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    ```bash
+    docker-compose up
+    ```
+5. Navigate to the `front-end` directory and serve the ember application.
+    > ❗ Make sure to include the proxy tag, otherwise the database cannot be contacted.
+    
+    >  🕓 Serving the application may take a while...
+    ```bash
+    ember s --proxy http://localhost:80
+    ```
```
Comment (juliavdkris) [inline README.md:39]: Perhaps mention that this is an optional step to make frontend dev a bit easier, but not actually "required" to get the project running (for a demo for example)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    
+3. In the `front-end` directory, install all npm dependencies
+   ```bash
+   cd ./front-end
+   npm install
+   ```
+4. Within the root of the project, run the Docker containers:
+    ```bash
+    docker-compose up
```
Comment (juliavdkris) [inline README.md:31]: This still needs the `FRONTEND_URL` and `BACKEND_URL` environment variables set!

For local development:
```
FRONTEND_URL=http://localhost
BACKEND_URL=http://localhost:8000
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+6. You can now access the application with the address the console returns. *(Normally, this should be http://localhost:4200)*
+
+### Server Deployment
+For the server deployment, please visit [this guide](https://github.com/osoc21/RoadBase/blob/master/misc/deployment/README.md).
+
+## Credits
+### Technical Team
+
+- Bo Robbrecht
+    - [Linked](https://www.linkedin.com/in/borobbrecht/)
```
Comment (juliavdkris) [inline README.md:49]: Typo ;)

```suggestion
    - [LinkedIn](https://www.linkedin.com/in/borobbrecht/)
```

File: README.md (hunk 1 above)
Comment (juliavdkris) [reply README.md:39]: Oh and proxy URL should be on port 8000, since that's where the API is running

**Review by juliavdkris — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (BT-Creator) [reply README.md:31]: Already added it in with the latest push. Thanks for the heads-up!

Comment (juliavdkris) [reply README.md:31]: Hahah saw your commit and discord message just after creating this review :p

File: README.md (hunk 1 above)
Comment (BT-Creator) [reply README.md]: Should be fixed now. There's now a extra section, dedicated for deployment, specific for development.

**Review by juliavdkris — APPROVED** [review]
Awesome!

**Pull request merged**